Doppler.ContactPolicies.Business.Logic/UserApiClient/Services/UserFeaturesService.cs

```diff
@@ -0,0 +1,44 @@
+using Doppler.ContactPolicies.Business.Logic.DTO;
+using Doppler.ContactPolicies.Business.Logic.Services;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Doppler.ContactPolicies.Business.Logic.UserApiClient.Services
+{
+    public class UserFeaturesService : IUserFeaturesService
+    {
+        private readonly IHttpClientFactory _clientFactory;
+        private readonly UserFeaturesServiceSettings _userFeaturesServiceSettings;
+        private readonly ILogger<UserFeaturesService> _logger;
+
+        public UserFeaturesService(IHttpClientFactory clientFactory, IOptions<UserFeaturesServiceSettings> userFeaturesServiceSettings, ILogger<UserFeaturesService> logger)
+        {
+            _clientFactory = clientFactory;
+            _userFeaturesServiceSettings = userFeaturesServiceSettings.Value;
+            _logger = logger;
+        }
+        public async Task<bool> GetUserContactPoliciesFeatureAsync(string accountName)
+        {
+            try
+            {
+                var baseUri = _userFeaturesServiceSettings.UsersApiURL;
+                var uri = new Uri(baseUri + $"/accounts/{accountName}/features");
+
+                var client = _clientFactory.CreateClient("users-api");
+                var responseString = await client.GetStringAsync(uri);
+
+                var features = JsonConvert.DeserializeObject<Features>(responseString);
```

Comment:
Si queres eliminar la dependencia a `Newtonsoft.Json`, podríamos usar lo siguiente:
```suggestion
                var features = JsonSerializer.Deserialize<Features>(responseString);
```

Reply:
gracias a tu comentario lo mejoré a esto:
` var features = await client.GetFromJsonAsync<Features>(uri);`

el cliente tenia un método para hacerlo directamente